src/renderer/mod.rs

```diff
@@ -39,7 +39,7 @@ pub struct RendererSettings {
     floating_blur_amount_y: f32,
     debug_renderer: bool,
     profiler: bool,
-    underline_automatic_scaling: bool,
+    underline_automatic_scaling: bool
```

Comment:
You might want to consider running `cargo fmt`, which will format the whole repo with the Rust community's defaults (which include a trailing comma). Or you set `g:rustfmt_autosave` to `v:true` directly.

Reply:
Fixed formatting with fmt.  Actually, I'm running it. Don't have fmt_autosave in my nvim config. Thanks for the remark.